test-cases/longevity/longevity-large-partition-3h.yaml

```diff
@@ -25,6 +25,7 @@ stress_cmd: ["scylla-bench -workload=sequential -mode=write -replication-factor=
              "scylla-bench -workload=sequential -mode=read  -replication-factor=3 -partition-count=25 -clustering-row-count=5555 -partition-offset=76   -clustering-row-size=uniform:1024..2048   -concurrency=100 -connection-count=100 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=170m -validate-data"
             ]
 
+post_prepare_cql_cmds: "CREATE INDEX si_text ON scylla_bench.test(pk) with comment = 'TEST INDEX'"
```

Comment:
Let's add it in a separate commit and add it also to other large partition test cases.
not all of them.
One with MV not only INDEX.

Reply:
The other large partition test has an MV already, hence why I only added an index to this one. I'll split the commit.